pkgdown/_pkgdown.yml

```diff
@@ -10,23 +10,15 @@ toc:
 template:
   bootstrap: 5
   bslib:
-    info: "#E6F2FD"
-    base_font:
-      google: {family: "Atkinson Hyperlegible", wght: [300, 400, 700], ital: [0, 1]}
-    headings-color: "#206B72"
-    link-color: "#216B73"
-    link-color-dark: "#cef4f9"
-    link-decoration: underline dotted
+    brand: pkgdown/_brand.yml
```

Comment:
pkgdown renders from the root folder, so we wouldn't need this if we had `_brand.yml` in the package root. (We wouldn't even need `bslib.brand`, but we could force it with `bslib.brand: true`.)

In our case, however, we don't want our example or test apps to accidentally pick up `_brand.yml`, so I've moved both it and `_pkgdown.yml` into `pkgdown/`. I can undo moving `_pkgdown.yml` if you want, but I generally prefer to keep all pkgdown things there.